modules/claims_api/app/controllers/concerns/claims_api/disability_compensation_validations.rb

```diff
@@ -43,6 +45,23 @@ def validate_form_526_submission_values!
       validate_form_526_treatments!
     end
 
+    def validate_form_526_current_mailing_address!
+      validate_form_526_current_mailing_address_country!
```

Comment:
why the extra step? could we just call `validate_form_526_current_mailing_address_country!` ?

Reply:
I imagine over time, we're going to need to add in validations for the other attributes in `currentMailingAddress`.
Right now, nothing in `currentMailingAddress` is being validated, BUT it's a required field.
So I imagine, we're gonna run into situations down the road where we find we have to validate other attributes in `currentMailingAddress`.  

Plus, it follows the same pattern already used for `validate_form_526_change_of_address!`.